app/models/document.rb

```diff
@@ -308,6 +308,15 @@ def self.slug
     title.parameterize.pluralize
   end
 
+  # Downcase but maintaining initials
+  def self.title_for_new_document
+    title.split(" ").map { |p| p.upcase == p ? p : p.downcase }.join(" ")
+  end
```

Comment:
It'd be nice to have a unit test for this. 

I also worry are there not more cases where this downcases something that should be title case? I see you got the statutory instrument so I'm guessing you went through them all.

Reply:
Yeah, I went through them all - and I updated the tests that already existed. I think it's hard to know from the code what should be downcased and not, so it's probably down to a content person to check when a new type is added. I'll add a unit test for the method.

Reply:
Unit tests added